# Part One

In [2]:
"12" * "3"

TypeError: can't multiply sequence by non-int of type 'str'

In [3]:
"3" + 3

TypeError: can only concatenate str (not "int") to str

In [4]:
int ("12.0")

ValueError: invalid literal for int() with base 10: '12.0'

In [5]:
num_pancakes = 10
"I am going to eat " + num_pancakes + " pancakes."

TypeError: can only concatenate str (not "int") to str

# Part Two

print(‘Hello World!”) ##Starts with single, ends with double \
print(“Hello World!”) ##No error \
print(“Hello World!’) ##Starts with a double, ends with a single \
print(‘How are you?) ##Starts with single, ends with ? \
print(“I’m fine’) ##Starts with double, contains single for apostrophe and ends with single \
print(‘’Of course,’ she said.’) ##Starts with single, contains inner single, ends with single \
print(‘“I’m fine,” she said.’)  ##Starts with single, contains inner double, ends with single

In [11]:
print("Hello World!")

Hello World!


In [12]:
print("Hello World!")

Hello World!


In [13]:
print("Hello World!")

Hello World!


In [14]:
print('How are you?')

How are you?


In [15]:
print("I’m fine")

I’m fine


In [17]:
print("Of course, she said.")

Of course, she said.


In [18]:
print("'I’m fine,' she said.")

'I’m fine,' she said.


# Part Three

In [19]:
Max

NameError: name 'Max' is not defined

In [20]:
int(‘five’)

SyntaxError: invalid character '‘' (U+2018) (<string>, line 1)

In [21]:
10/0.0

ZeroDivisionError: float division by zero

In [22]:
in = ‘input1.shp’

SyntaxError: invalid syntax (<string>, line 1)

In [23]:
‘five’ + 6

SyntaxError: invalid character '‘' (U+2018) (<string>, line 1)

In [24]:
1st_input = ‘nlcd2010.shp’

SyntaxError: invalid syntax (<string>, line 1)

Max - NameError: name 'Max' is not defined \
int(‘five’) - SyntaxError: invalid character '‘' (U+2018) (<string>, line 1) \
10/0.0 - ZeroDivisionError: float division by zero \
in = ‘input1.shp’ - SyntaxError: invalid syntax (<string>, line 1) \
‘five’ + 6 - SyntaxError: invalid character '‘' (U+2018) (<string>, line 1) \
1st_input = ‘nlcd2010.shp’ - SyntaxError: invalid syntax (<string>, line 1)

In [33]:
course_number = "gis 4653"
print(course_number.replace('4', '5'))
print(course_number.upper())
print(course_number[4:])

gis 5653
GIS 4653
4653


What is printed to the screen after ALL the following statements are executed?

4653 

What is the value of the string variable course_number  after ALL of the following statements are executed?

gis 4653 \
gis 5653 \
GIS 4653 \
4653 

Is this the result you expected? Explain why/why not. \

Yes, I have coding experience and have a strong understanding of variables and state.